content/millennium/bulk-data/group-export.md

```diff
@@ -0,0 +1,273 @@
+---
+title: Group Export | R4 API
+---
+
+# Group Export
+
+<%= beta_tag %>
+
+* TOC
+{:toc}
+
+## Overview
+
+The Group Export operation is used to obtain FHIR resources for all members of a specified group of patients.
+
+An export process uses the [FHIR Asynchronous Request Pattern](https://hl7.org/fhir/R4/async.html) to kick off the export job for a group of patients. Once the export job is kicked off, the status endpoint may be called to check the progress of the job. Once the job is complete, the status endpoint returns a list of files which may be downloaded using the file retrieval endpoint. After the appropriate ndjson files are downloaded, the delete endpoint should be called to clean up the no longer needed files.
+
+The Group Export operation exports resources required for [United States Core Data for Interoperability V1 (USCDI)](https://www.healthit.gov/isa/united-states-core-data-interoperability-uscdi). For a full list of resources see the [US Core Profiles](http://hl7.org/fhir/us/core/STU4/#us-core-profiles). When the `_type` parameter is used it returns only the FHIR resources specified. The `_type` parameter should be used to filter the resources required for a specific use case. Limiting the number of requested resources in the `_type` parameter is recommended as it can decrease response times.
+
+### Recommendations
+
+* Limit group sizes to around a thousand patients or fewer.
```

Comment:
just putting a note here that we may need to remove this - we'll discuss next week with Andrew

Reply:
> just putting a note here that we may need to remove or update this limit - we'll discuss next week with Andrew

@SA030800  @MadhurRaj  @ebbp123  In Mikes illumination to clients, he has a recommendation for limiting the group to 10,000 patients or fewer. This is more of a recommendation rather than a hard limit. Seeing that we are getting ready to publish this soon, should we go ahead and get this 'recommendation' update to ten thousand?

Reply:
how about we publish the limit later and remove this statement. Anyways  the recommendation is so that they can get a faster export. As of now they cannot great a group so this info is not useful

Reply:
> how about we publish the limit later and remove this statement. Anyways the recommendation is so that they can get a faster export the limit is if they go beyond that it is an error case. As of now they cannot create a group so this info is not useful

I'm okay with removing it for now and adding it back in later.

Reply:
I have removed the recommendation: https://github.com/cerner/fhir.cerner.com/pull/899/commits/92f958fceee83a8e908628b53f8ed6480d856941#diff-c2a4803bbbf3b17e514d41cc6eb7d578c94a283f937cce0400f5437570744d7fL24